lib/xenopsd.ml

```diff
@@ -55,6 +56,7 @@ let options = [
     "default-vbd-backend-kind", Arg.Set_string default_vbd_backend_kind, (fun () -> !default_vbd_backend_kind), "Default backend for VBDs";
     "ca-140252-workaround", Arg.Bool (fun x -> ca_140252_workaround := x), (fun () -> string_of_bool !ca_140252_workaround), "Workaround for evtchn misalignment for legacy PV tools";
     "additional-ballooning-timeout", Arg.Set_float additional_ballooning_timeout, (fun () -> string_of_float !additional_ballooning_timeout), "Time we allow the guests to do additional memory ballooning before live migration";
+    "vif_ready_timeout", Arg.Set_int vif_ready_timeout, (fun () -> string_of_int !vif_ready_timeout), "Time before we assume vif has connected";
```

Comment:
Please make this specific to IGMP querying, otherwise no one will understand what this is. Also, please use `-` in the string, not `_`.